docs/manage/_lessons.rst

```diff
@@ -8,47 +8,54 @@ Kolibri **Lessons** offer you the possibility to prepare smaller selections of a
 
 To view, create or delete lessons, as well as assign them to learners, use the **Lessons** tab in your **Coach** dashboard tab (|lessons| icon). Default view displays the list of all lessons created for the selected class, the number of resources they contain, groups they are assigned to, and their status indicator. You can filter the lessons in order to see all, or just active/inactive ones. 
 
-.. image:: img/lessons.png
-  :alt: manage lessons home page
+  .. figure:: img/lessons.png
+    :alt: 
 
 .. note::
-  To manage **Lessons** in Kolibri classes and groups, you must sign in as **Coach** or **Admin**.
+  To manage **Lessons** in Kolibri classes and groups, you must sign in as **coach**, **admin** or **super admin**.
 
 
 Create New Lesson
 -----------------
 
 To create a new lesson, follow these steps.
 
-1. Click the **New lesson** button.
-2. Fill in the field for the lesson title, description and set the visibility. 
+1. Click the :guilabel:`NEW LESSON` button.
+2. Fill in the field for the lesson title, description and choose the group(s) or the entire class as the recipients. 
 
-  .. image:: img/new-lesson.png
-    :alt: create new lesson options
+  .. figure:: img/new-lesson.png
+    :alt: 
 
-3. Click the **Continue** button. In this step, the lesson does not have any learning resources yet, but you can see and edit some options.
+3. Click :guilabel:`CONTINUE`. In this step, the lesson does not have any learning resources yet, but you can see and edit some options.
 
-  .. image:: img/lesson-options.png
-    :alt: new lesson with no resources added
+  .. figure:: img/lesson-options.png
+    :alt: New lesson with no resources added; navigate to Options button and click to open it for more actions.
 
-4. Click the **Add resources** button, and select the content channel from which you wish to select resources for the lesson.
+4. Click :guilabel:`MANAGE RESOURCES` and select the content channel from which you wish to select resources for the lesson.
 
-  .. image:: img/select-lesson-resources.png
-    :alt: add resources to the lesson
+  .. figure:: img/select-lesson-resources.png
+    :alt: This window presents a list of channels from which you can choose to add resources to the lesson.
 
-5. Keep navigating the topic tree of the selected content channel until you arrive to single content items. Use the checkboxes to select the resources you want to include in the lesson.
+5. Keep navigating the topic tree of the selected content channel until you arrive to single content items. Use the checkboxes to select the resources you want to include in the lesson, and you will see a snackbar confirmation at the bottom that the resource has been added.
```

Comment:
> snackbar confirmation 

This is an internally-used term but I don't think it's something that most users will by familiar with. Maybe "pop-up confirmation"

Reply:
Yes, I agree that the term was not the most appropriate, and I've reworded the sentence :slightly_smiling_face: